### PR TITLE
feat: --no-color to disable colored output

### PR DESCRIPTION
**Pull request by ANGkeith**
image

add options for disable color, 

main motivation is to simplify our integration tests; no need to add `chalk` or stripAnsi

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -269,6 +269,16 @@ process.on("SIGUSR2", async () => await cleanupJobResources(jobs));
...
+        .option("color", {
+            requiresArg: false,
+            default: false,
+            // placeholder for the --no-color option
+            hidden: true,
+        })
+        .option("no-color", {
+            requiresArg: false,
+            description: "Disable all color.",
+        })
```
Comment (firecow) [inline src/index.ts:281]: This is redundant
```suggestion
```

Comment (ANGkeith) [reply src/index.ts:281]: hacky, but added that in for the --help message haha

not sure if there's a nicer way for it

File: src/index.ts [hunk 2]
```diff
@@ -269,6 +269,16 @@ process.on("SIGUSR2", async () => await cleanupJobResources(jobs));
...
         })
+        .option("color", {
+            requiresArg: false,
+            default: false,
+            // placeholder for the --no-color option
+            hidden: true,
+        })
+        .option("no-color", {
+            requiresArg: false,
+            description: "Disable all color.",
```
Comment (ANGkeith) [inline src/index.ts:280]:
```suggestion
            description: "Disable all color",
```

File: src/index.ts [hunk 3]
```diff
@@ -269,6 +269,16 @@ process.on("SIGUSR2", async () => await cleanupJobResources(jobs));
             description: "The name, without the architecture, of a gitlab hosted runner to emulate. See here: https://docs.gitlab.com/ee/ci/runners/hosted_runners/linux.html#machine-types-available-for-linux---x86-64",
             choices: GitlabRunnerPresetValues,
         })
+        .option("color", {
+            requiresArg: false,
+            default: false,
+            // placeholder for the --no-color option
+            hidden: true,
+        })
+        .option("no-color", {
```
Comment (ANGkeith) [inline src/index.ts:278]:
```suggestion
        // NOTE: not redundant, this is added for the `--help` message
        .option("no-color", {
```

Comment (firecow) [reply src/index.ts:279]: It is redundant until we decide to properly add `--no` versions for every boolean values in index.ts

image

File: src/index.ts [hunk 4]
```diff
@@ -269,6 +269,17 @@ process.on("SIGUSR2", async () => await cleanupJobResources(jobs));
...
+            requiresArg: false,
+            default: false,
+            // placeholder for the --no-color option
+            hidden: true,
+        })
+        // NOTE: not redundant, this is added for the `--help` message
+        .option("no-color", {
+            requiresArg: false,
+            description: "Disable all color",
+        })
```
Comment (firecow) [inline src/index.ts:282]: This is redundant. There is a description in the top of help showing all commands can be negated with `--no`
```suggestion
```

Comment (ANGkeith) [reply src/index.ts]: ahh i see your point, i just noticed the description haha

**Review by firecow — APPROVED** [review]

**Pull request merged**